In [1]:
import pandas as pd

df_flights = pd.read_csv("../data/processed/cleaned_flights.csv")
df_hotels = pd.read_csv("../data/processed/cleaned_hotels.csv")
df_trends = pd.read_csv("../data/processed/cleaned_trends.csv")

display(df_flights["price"].describe())

count      501.000000
mean      3415.772455
std       1642.152805
min       1913.000000
25%       2564.000000
50%       2900.000000
75%       4076.000000
max      14096.000000
Name: price, dtype: float64

- The median (2900) does not exceed the mean (3415.77), indicating that while most flight prices are below average, there are a number of high-priced flights dragging the price up.
- The standard deviation (1642.15) is almost half of the mean (3415.77), indicating some relative dispersion and some volatility in flight rates.
- Q2 of flight prices is below the mean, showing half of the values are still below average, while Q3 shows that the upper range of the values are above, but still relatively close to the median, indicating a few expensive flights are dragging up the price.
- The inter‑quartile range (1512) is more than half the median, indicating a wide dispersion of prices in the middle 50% of the flight listings.
- Using the Tukey rule, the upper out‑lier boundary is 6344, which is above Q3. With a maximum of 14096, this indicates that there are a few high-priced flights as outliers.

In [2]:

display(df_hotels["price_php"].describe())

count     872.000000
mean     2340.698394
std      1342.193081
min       661.000000
25%      1493.750000
50%      1833.000000
75%      2856.000000
max      7403.000000
Name: price_php, dtype: float64

- The median (1833) does not exceed the mean (2340), indicating a right‑skewed (positively skewed) distribution: most hotels have relatively below average nightly rate, while a few hotels drag the average up.
- The standard deviation (1,342.19) is more than half of the mean (2,340.70), indicating a high relative dispersion and substantial volatility in nightly hotel rates.
- Q2 of hotel prices is below the mean, showing half of the values are still below average, while Q3 shows that the upper range of the values are above, but still relatively close to the median, indicating a few expensive hotels are dragging up the price.
- The inter‑quartile range (1362.25) is substantial relative to the median, indicating a wide dispersion of prices in the middle 50% of the hotel market.
- Using the Tukey rule, the upper out‑lier boundary is 4899.375, which is above Q3. With an observed maximum of 7403, this indicates that there are a few high-priced hotel stays as outliers.

In [3]:

display(df_trends["value"].describe())

count    2046.000000
mean       27.734115
std        28.716925
min         0.000000
25%         0.000000
50%        35.000000
75%        51.000000
max       100.000000
Name: value, dtype: float64

- The median (35) exceeds the mean (27.7), indicating a left‑skewed (negatively skewed) distribution: most days have relatively high interest scores, while a few low‑interest days drag the average down.
- The standard deviation (28.7) is almost as large as the mean, showing that Google‑Trends scores are highly variable from week to week.
- One‑quarter of the days have a trend score of 0, so low‑interest days are common. The maximum of 100 occurs only in the upper tail of the distribution.
- The inter‑quartile range (51) covers more than half of the possible score range (0‑100), indicating that the middle 50 % of days are spread across a broad spectrum of interest levels.
- Using the Tukey rule, the upper out‑lier boundary is 127.5, which is beyond the observed maximum (100). Hence the data contain no extreme high outliers.